README.md

```diff
@@ -4,13 +4,29 @@
 
 Feathr lets you:
 
-- **define features** based on raw data sources, including time-series data, using simple APIs.
-- **get those features by their names** during model training and model inferencing.
-- **share features** across your team and company.
+- **Define features** based on raw data sources, including time-series data, using simple APIs.
+- **Get those features by their names** during model training and model inferencing.
+- **Share features** across your team and company.
 
 Feathr automatically computes your feature values and joins them to your training data, using point-in-time-correct semantics to avoid data leakage, and supports materializing and deploying your features for use online in production.
 
-## Installation
+## Running Feathr on Azure with 3 Simple Steps
+
+Feathr has native cloud integration. To use Feathr on Azure, you only need three steps:
+
+1. Get the principal ID of your account by running `az ad signed-in-user show --query objectId -o tsv` in the link below (Select "Bash" if asked), and write down that value (something like `b65ef2e0-42b8-44a7-9b55-abbccddeefff`)
+
+[Launch Cloud Shell](https://shell.azure.com/bash)
+
+2. Click the button below to deploy a minimal set of Feathr resources for demo purpose.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Flinkedin%2Ffeathr%2Fone_click_deployment%2Fdocs%2Fhow-to-guides%2Fazure_resource_provision.json)
+
+3. Run the Feathr Jupyter Notebook by clicking the button below. You only need to fill in the resource prefix.
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/linkedin/feathr/main?labpath=feathr_project%2Ffeathrcli%2Fdata%2Ffeathr_user_workspace%2Fnyc_driver_demo.ipynb)
+
+## Feathr Installation
```

Comment:
How `Feathr Installation` is related to the one-click? This is confusing and why do i need it?

It's probably can go into sub folders or deserve better explanation.

Reply:
updated with clear title